config.yml

```diff
@@ -1,6 +1,15 @@
 specs:
-  1.14: https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.14/api/openapi-spec/swagger.json
-  1.15: https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.15/api/openapi-spec/swagger.json
-  1.16: https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.16/api/openapi-spec/swagger.json
-  1.17: https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.17/api/openapi-spec/swagger.json
-  1.18: https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.18/api/openapi-spec/swagger.json
+
+# Kubernetes core APIs
+- output: 'gen/local'
+  openapi: http://localhost:8001/openapi/v2
+  prefix: io.k8s.api
+  patchDir: custom/core
+  extensionDir: extensions/core
```

Comment:
Generally happy with this! Could you re-include the Kubernetes swagger urls for 1.14 to 1.18, so that default behavior of this repository stays the same?

For CRD libraries, we could instead create new repositories on GitHub and have both the respective config and the artifacts in there. wdty?